<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphWindow</class>
 <widget class="QMainWindow" name="GraphWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1918</width>
    <height>917</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GraphWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1641</width>
      <height>841</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="locButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>500</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="locButton_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>830</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="locButton_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>830</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="locButton_4">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>840</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="locButton_5">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>850</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="locButton_6">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>860</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="locButton_7">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>870</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="locButton_8">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>880</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="locButton_9">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>890</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="locButton_10">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>900</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="locButton_11">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>910</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>11</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="locButton_12">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>910</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>12</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="locButton_13">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>910</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>13</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="locButton_14">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>910</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>14</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="locButton_15">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>910</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>15</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="locButton_16">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>910</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>16</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="locButton_17">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>910</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>17</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1740</x>
      <y>10</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>当前地点</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>40</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>地点名字：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>60</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>地点简介：</string>
    </property>
   </widget>
   <widget class="QLabel" name="locName">
    <property name="geometry">
     <rect>
      <x>1740</x>
      <y>40</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="locDesc">
    <property name="geometry">
     <rect>
      <x>1740</x>
      <y>60</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>130</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新地图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sButton">
    <property name="geometry">
     <rect>
      <x>1800</x>
      <y>130</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>选择起点</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tButton">
    <property name="geometry">
     <rect>
      <x>1800</x>
      <y>220</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>选择终点</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mstButton">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>220</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>全景点路径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="allPath">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>310</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>所有路径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bestPath">
    <property name="geometry">
     <rect>
      <x>1800</x>
      <y>310</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>选择最优路径</string>
    </property>
   </widget>
   <widget class="QLabel" name="disLabel">
    <property name="geometry">
     <rect>
      <x>1690</x>
      <y>590</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0m</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1918</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
